sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -272,8 +282,17 @@ public <K, V> KafkaConsumer<K, V> createConsumer(
           org.apache.kafka.clients.consumer.ConsumerConfig.CLIENT_ID_CONFIG,
           consumerConfig.getClientId() + "-" + instanceId);
     }
+    KafkaConsumer<K, V> consumer =
+        new KafkaConsumer<>(consumerProperties, keyDeSerializer, valueDeSerializer);
+
+    if (micrometerConsumerListener != null) {
+      micrometerConsumerListener.consumerAdded(
+          consumerProperties.getProperty(
+              org.apache.kafka.clients.consumer.ConsumerConfig.CLIENT_ID_CONFIG),
```

Comment:
Why don't we use the `clientIdOptional` from above?

Reply:
The `clientIdOptional` is not available at this point. Actually the `clientIdOptional` is just a complicated way to read out the value that is set here. Sadly the KafkaConsumer has no method to access that value. There is most likely a more elegant way to handle that value but i have not put effort in looking into this without changing public method signatures